src/content/docs/fr/guides/deploy/render.mdx

```diff
@@ -19,7 +19,7 @@ Vous pouvez déployer votre projet Astro sur [Render](https://render.com/), un s
 
 4. Donnez un nom à votre site web, sélectionnez la branche et spécifiez la commande de construction et le répertoire de publication
    - **build command:** `npm run build` (Commande de construction)
-   - **publish directory:** `dist` (Répertoire de publication)
+   - **publish directory:** `dist` pour les sites statiques; `dist/client` si vous utilisez les modes d'affichage serveur `server` ou `hybrid`. (Répertoire de publication)
```

Comment:
```suggestion
   - **publish directory:** `dist` pour les sites statiques ; `dist/client` si vous utilisez les modes d'affichage serveur `server` ou `hybrid`. (Répertoire de publication)
```
(In French, we have an extra space before the semi-colon)

Optionally, another suggestion because "*modes d'affichage serveur*" bothers me a little. I would suggest "*modes d'affichage côté serveur*" or "*modes de rendu côté serveur*". But feel free to leave it as it is if you find that adding "*côté*" does not improve the sentence.

```suggestion
   - **publish directory:** `dist` pour les sites statiques ; `dist/client` si vous utilisez les modes de rendu côté serveur `server` ou `hybrid`. (Répertoire de publication)
```